core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -92,6 +92,12 @@ export class TopicEditorTabComponent implements OnInit, OnDestroy {
   uncategorizedEditOptionsIndex: number;
   subtopicEditOptionsAreShown: number;
   skillOptionDialogueBox: boolean = true;
+  MAX_CHARS_IN_TOPIC_NAME!: number;
```

Comment:
This format (snake upper case) is almost always reserved for constants, not fields. Let's use a camel case with fields here.

Reply:
Ok! updated.